modules/cluster-logging-deploy-clo.adoc

```diff
@@ -1,86 +1,27 @@
-in {product-title} for installing the operators individually.
-You *must* install the Elasticsearch Operator using the CLI following the directions below.
-You can install the Cluster Logging Operator using the web console or CLI.
-====
+. Create a Namespace for the Cluster Logging Operator:
 
-.Prerequisites
+.. In the {product-title} web console, click *Home* -> *Projects*.
 
-Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
-requires its own storage volume.
+.. Click *Create Project*.
 
-Elasticsearch is a memory-intensive application. Each Elasticsearch node needs 16G of memory for both memory requests and limits.
-The initial set of {product-title} nodes might not be large enough to support the Elasticsearch cluster. You must add additional nodes to the
-{product-title} cluster to run with the recommended or higher memory. Each Elasticsearch node can operate with a lower
-memory setting though this is not recommended for production deployments.
+.. Enter `openshift-logging` in the *Name* and *Display Name* fields.
 
-[NOTE]
-====
-You *must* install the Elasticsearch Operator using the CLI following the directions below.
-You can install the Cluster Logging Operator using the web console or CLI.
-====
+.. Optional: Enter a description.
 
-.Procedure
-
-. Create Namespaces for the Elasticsearch Operator and Cluster Logging Operator.
-+
-[NOTE]
-====
-You can also create the Namespaces in the web console using the *Administration* -> *Namespaces* page.
-You must apply the `cluster-logging` and `cluster-monitoring` labels listed in the sample YAML to the namespaces you create.
-====
+.. Click *Create*.
 
-.. Create a Namespace for the Elasticsearch Operator (for example, `eo-namespace.yaml`):
-+
-[source,yaml]
-----
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: openshift-operators-redhat <1>
-  annotations:
-    openshift.io/node-selector: ""
-  labels:
-    openshift.io/cluster-monitoring: "true" <2>
-----
-<1> You must specify the `openshift-operators-redhat` namespace. To prevent 
-possible conflicts with metrics, you should configure the Prometheus Cluster 
-Monitoring stack to scrape metrics from the `openshift-operators-redhat` 
-namespace and not the `openshift-operators` namespace. The `openshift-operators` 
-namespace can contain community operators, which are untrusted and could publish 
-a metric with the same name as an {product-title} metric, which would cause 
-conflicts. 
-<2> You must specify this label as shown to ensure that cluster monitoring 
-scrapes the `openshift-operators-redhat` namespace.
-
-.. Run the following command to create the namespace:
-+
-----
-$ oc create -f <file-name>.yaml
-----
-+
-For example:
-+
-----
-$ oc create -f eo-namespace.yaml
-----
+.. On the *Project Details* page, click *YAML*.
 
-.. Create a Namespace for the Cluster Logging Operator (for example, `clo-namespace.yaml`):
+.. Replace the YAML shown with the following:
```

Comment:
Usage of "shown" here is confusing. I get that it means the YAML shown on the Project Details page, but it took me a minute. Maybe, "Replace the YAML shown on the page with the YAML in the following example"?

Reply:
outdated